src/app/loans/loans-view/transactions/view-transaction/view-transaction.component.html

```diff
@@ -4,8 +4,8 @@
     <button mat-raised-button color="primary" [routerLink]="'edit'" *ngIf="allowEdition">
       <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;Edit
     </button>
-  
-    <button mat-raised-button color="warn" *ngIf="allowEdition" (click)="undoTransaction()">
+
+    <button mat-raised-button color="warn" (click)="undoTransaction()">
```

Comment:
If you are not allowed to edit, you are not allowed to undo it either. Am I wrong?

Reply:
Not necessary, I mean you can to have a Goodwill credit transaction that can not be edited but It can be undone